tssc/step_implementers/push_container_image/skopeo.py

```diff
@@ -56,9 +88,9 @@ def _validate_step_config(self, step_config):
     def _run_step(self, runtime_step_config):
 
         version = "latest"
-        if(self.get_step_results('generate-metadata') and \
-          self.get_step_results('generate-metadata').get('image-tag')):
-            version = self.get_step_results('generate-metadata')['image-tag']
+        if(self.get_step_results(DefaultSteps.GENERATE_METADATA) and \
```

Comment:
this is WAY better. +1 good idea.

Reply:
:)

there are still other cases i didn't catch, but i saw this one and had to fix it.